prompterator/main.py

```diff
@@ -411,6 +427,18 @@ def set_up_ui_generation():
         label_visibility="collapsed",
         key=c.PROMPT_COMMENT_KEY,
     )
+    selected_model: c.ModelProperties = st.session_state.model
+    available_structured_output_settings = selected_model.supports_structured_output
+    structured_output_available = False
+    if len(available_structured_output_settings) > 1:
```

Comment:
@gkaretka is checking on `len` the best option there is? It does read a bit fishy for my liking.